docs/website/mtouch-errors.md

```diff
@@ -2545,6 +2545,14 @@ If this is not the case, please file a [bug report](https://github.com/xamarin/x
 
 The name of an Objective-C protocol can't contain certain characters which means that the `Adopts` attribute on the corresponding managed class can't have the `ProtocolType` parameter containing them. Please refer to the provided error message and fix accordingly.
 
+### MT4178: The class '{class}' will not be registered because the WatchKit framework has been removed from iOS.
+
+Apple removed the WatchKit framework on iOS, so any code that uses it will not work anymore.
```

Comment:
That seems inaccurate as introspection did not complain. Only the headers (not the framework) was removed (at least so far)